lib/dynflow/throttle_limiter.rb

```diff
@@ -0,0 +1,105 @@
+module Dynflow
+  class ThrottleLimiter
```

Comment:
Wouldn't it be odd to have something like `DelayedExecutor::ThrottleLimiter` which would not extend `DelayedExecutor::Abstract`? However I wouldn't mind moving it from the `::Dynflow` scope